lighthouse-core/fraggle-rock/gather/base-gatherer.js

```diff
@@ -90,6 +90,9 @@ class FRGatherer {
    * @return {Promise<LH.Gatherer.PhaseResultNonPromise>}
    */
   async afterPass(passContext, loadData) {
+    if ('dependencies' in this.meta) {
```

Comment:
If we really want to use `if (this.meta.dependencies)` instead, we can add a `@ts-ignore` or add `dependencies?: {}` to the type definition of `GathererMetaNoDependencies`.

Reply:
I'm not sure that's necessary, this seems great :)